deploy/runtools/run_farm_deploy_managers.py

```diff
@@ -646,25 +812,35 @@ def terminate_instance(self) -> None:
 
 class VitisInstanceDeployManager(InstanceDeployManager):
     """ This class manages a Vitis-enabled instance """
+    
+    @classmethod
+    def sim_command_requires_sudo(cls) -> bool:
+        """ This sim does not require sudo. """
+        return False
+
     def __init__(self, parent_node: Inst) -> None:
         super().__init__(parent_node)
 
+        # Vitis runs add the additional handling of the xclbin URI
+        self.uri_list.append(URIContainer('xclbin', self.get_xclbin_name()))
+
     def clear_fpgas(self) -> None:
         if self.instance_assigned_simulations():
             self.instance_logger("""Clearing all FPGA Slots.""")
 
             card_bdfs = []
             with settings(warn_only=True), hide('everything'):
-                temp_file = "/tmp/xbutil-examine-out.json"
-                collect = run(f"xbutil examine --format JSON -o {temp_file}")
-                with open(temp_file, "r") as f:
-                    json_dict = json.loads(f.read())
+                # Examine forcibly puts the JSON in an output file (on the remote); The stdout
+                # it produces is difficult to parse so use process substitution
+                # to pipe JSON to stdout instead.
+                xbutil_examine_json = run("xbutil examine --force --format JSON -o >(cat) > /dev/null")
+                json_dict = json.loads(xbutil_examine_json)
```

Comment:
I can see how they were writing to a hardcoded tempfile on a possibly remote host and then opening it without copying it back.  In this new way of doing things, anything `xbutil` emits into `STDERR` could possibly cause difficult to understand and diagnose JSON parsing errors.

It better to pass `combine_stderr=False` (see [fabric docs](https://docs.fabfile.org/en/1.14/usage/interactivity.html#combine-streams))  to this invocation of `run()` and DEBUG log `xbutil_examine_json.stderr` if it isn't empty.

I don't remeember what uncaught `JSONDecodeError` stack traces look like but I vaguely remember them being unhelpful when the JSON being parsed is dynamically generated because they tell you where the error happened but don't print out what was parsed.  You might want to DEBUG log `xbutil_examine_json` or catch `JSONDecodeError` to raise our own meaningful `RuntimeError` message that includes the `JSONDecodeError.doc` in it's message